introduction/de/06-conclusion.asciidoc

```diff
@@ -0,0 +1,13 @@
+== Schusswort
+
+Auf dieser Serie haben wir eine Einführung in InnerSource gegeben.
+InnerSource wendet Open Source Praktiken und Prinzipien auf die firmeninterne Softwareentwicklung an.
+Es bietet Nutzern eine zusätzliche Option, speziell wenn Produktionsteams nicht in der Lage sind eine erforderliche Funktionsanforderung im notwendingen Zeitraum zu liefern.
+Erfolgreiche InnerSource umfasst https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/product-owner/01-opening-article.asciidoc[_Produkt Owner_] und https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-Committer/01-Introduction.asciidoc [_Trusted Committer_] vom *Hostteam* sowie ein https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc [_Contributor_] vom *Gastteam*.
+Richtig betriebene InnerSource bietet beiden teilnehmenden Teams viele Vorteile.
+Die Schlüsselprinzipien, nach denen gut geführte InnerSource funktioniert, sind *freiwilliger Code-Beitrag* und *priorisierte Schulung/Betreuung*.
+
+Während diese Serie einen allgemeinen Überblick über InnerSource enthält, gibt es viele weitere wichtige Details um InnerSource in Ihrem Team erfolgreich zu praktizieren.
+Wenn Sie mit den Entwicklungen in InnerSource und seine besten Praktiken auf dem laufenden bleiben möchten, laden wir Sie herzlich zu den http://innersourcecommons.org [dem InnerSource Commons] ein.
+Weiterhin betreibt die InnerSource Commons einen Slack-Kanal, eine InnerSource Arbeitsgruppe die sich mit erfolgreichen Zusammenarbeitsmustern befasst, und veranstaltet jedes Jahr mehrere Treffen.
```

Comment:
```suggestion
Weiterhin betreibt die InnerSource Commons Foundation einen Slack-Kanal, sowie eine InnerSource Working Group die sich mit erfolgreichen InnerSource Patterns befasst, und veranstaltet jedes Jahr mehrere Treffen.
```

Reply:
This change would need to be ported to the English version as well.   Should we just leave this PR as a strict translation of the English content and then have a separate PR for adjusting content?

Reply:
+1